docs/rules/no-html-safe.md

```diff
@@ -0,0 +1,91 @@
+# no-html-safe
+
+`htmlSafe` marks a string as safe for unescaped output with Ember templates so you can render it as HTML. While useful this can inadvertently open you up to Cross-site Scripting (XSS) vulnerabilities, especially if the string was generated from user input or some other untrusted source. **You should only ever use `htmlSafe` with trusted or sanitized input**.
+
+## Rule Details
+
+This rule prevents importing the `htmlSafe` utillity from `@ember/template` (or `@ember/string` for older Ember versions);
+
+## Examples
+
+Examples of **incorrect** code for this rule:
+
+```js
+import { htmlSafe } from '@ember/template';
```

Comment:
I'm not entirely sure how `htmlSafe` is available from both `@ember/string` and `@ember/template`. The [Ember API docs](https://api.emberjs.com/ember/release/functions/@ember%2Ftemplate/htmlSafe) say it comes from `@ember/template` but we import it from `@ember/string` in our app 🤷‍♂️.

Reply:
The location of it was changed to `@ember/template` recently. So catching both makes sense.